treeherder/etl/artifact.py

```diff
@@ -149,8 +149,8 @@ def store_job_artifacts(artifact_data):
                                    "means the job was already parsed",
                                    job_guid)
             else:
-                logger.warning("Unknown artifact type: %s submitted with job %s",
-                               artifact_name, job.guid)
+                logger.info("Unknown artifact type: %s submitted with job %s",
```

Comment:
Maybe this should stay as a warning. Perhaps @camd can weigh in as to whether this message has ever been useful in papertrail for troubleshooting.

Reply:
I was thinking a bit more about this.

The original intent of the bug is two-fold:

1. Change our Papertrail logging to show INFO messages
    - Because there are a lot of code (ours and from packages) that use info by standard even though some of them should be warnings
    - Thus, such messages can be missed and it happened to me when I was try to find problems in production
2. I noticed some warning messages that should have been info messages
    - I noticed that Ionut (but anyone of us would have wanted to do the same) had some warning messages just to make them show up on Papertrail [1]
    - Only those messages should be lowered to info.

I believe we should focus on just the first point (WARNING -> INFO) and simply fix output in the future as we work on other features. We, the long-term contributors, have the context to decide which messages are clearly just informative messages versus warning ones.

@sarah-clements @camd does this work for you? What do you think?

[1]
```
treeherder/perf/alerts.py:            self.log.warning('Found small backfill range (of size {} instead of {})'
treeherder/perf/alerts.py:                self.log.warning(f"Failed to compute report for alert summary {summary}. {ex}")
treeherder/perf/models.py:            logger.warning('Removing performance signatures with missing jobs...')
treeherder/perf/models.py:            logger.warning('Exception: {}'.format(ex))
treeherder/perf/models.py:            logger.warning(ex)
treeherder/perf/models.py:                    logger.warning('Failed to delete performance data chunk, while running "{}" query '
treeherder/perf/models.py:                        logger.warning('Successfully deleted {} performance datum rows'.format(deleted_rows))
```